controller/endpointcontroller.php

```diff
@@ -70,7 +70,7 @@ public function get() {
 		// We stop polling for them.
 		if (!$this->manager->hasNotifiers()) {
 			$response = new Response();
-			$response->setStatus(Http::STATUS_NOT_FOUND);
+			$response->setStatus(Http::STATUS_NO_CONTENT);
 			return $response;
```

Comment:
We use the 204 to stop asking for notifications, when there is no app registered that generates one (default in 8.2)

There is no need to poll every 30 seconds, when nothing can ever potentially show up.